op-node/rollup/derive/deposit_log_test.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 func TestUnmarshalLogEvent(t *testing.T) {
+	t.Skip("not working because deposit_log_create not working properly")
```

Comment:
When unskipped, this test fails with: 
`/Users/maurelian/Projects/O/mono/op-node/rollup/derive/deposit_log_test.go:29: expected 4 event topics (event identity, indexed from, indexed to, indexed version), got 3`

Reply:
To fix, `MarshalDepositLogEvent` will need to return the version on the `Topics` field.

https://github.com/ethereum-optimism/optimism/blob/19490d318/op-node/rollup/derive/deposit_log.go#L119-L156